metrics/collectors/network.lua

```diff
@@ -0,0 +1,22 @@
+local metrics = require('metrics.details')
+
+
+local function init()
+    local net_sent_total = metrics.Gauge.new('tnt_net_sent_total', 'Totally sent in bytes')
+    local net_sent_rps = metrics.Gauge.new('tnt_net_sent_rps', 'RPS on sent')
+    local net_received_total = metrics.Gauge.new('tnt_net_received_total', 'Totally received in bytes')
+    local net_received_rps = metrics.Gauge.new('tnt_net_received_rps', 'RPS on receive')
+
+    global_metrics_registry:register_callback(function()
+        local box_stat_net = box.stat.net()
```

Comment:
Consider also the new fields ( https://github.com/tarantool/doc/issues/760 ) as optional ones.

Reply:
added in eae06d527020c0f0b94cb4e930f730c4c3a43f9b

Reply:
f5bb13768ec7f9c3ef96d267509de8024dc05dc5

changed help for some of new metrics